### PR TITLE
App: add missing translations

### PR DESCRIPTION
**Pull request by Botsy**
In this PR all hardcoded strings in web app are replaced with translation keys that were added in stremio-translations. A script for locating the hardcoded strings is added, but could be improved in future to detect more precisely all edge cases;

### CODE REVIEW

File: scan-i18n-report.js
Comment (tymmesyde) [inline scan-i18n-report.js:1]: Can you convert this to a test instead without writing a csv file ?

File: src/common/useTranslate.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ const useTranslate = () => {
     const string = useCallback((key) => t(key), [t]);
 
     const stringWithPrefix = useCallback((value, prefix, fallback = null) => {
-        const key = `${prefix}${value}`;
+        const key = `${prefix || ''}${value}`;
```
Comment (tymmesyde) [inline src/common/useTranslate.js]: The point of this function is to use a prefix so it can't be an empty string

File: tests/i18nScan.test.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+            console.log('🚨 Hardcoded strings found:');
+            report.forEach((row) => {
+                console.log(`File: ${row.file}, Line: ${row.line}, String: "${row.string}", Suggested Key: ${row.key}`);
+            });
+        }
+
+        // Optional: expect no hardcoded strings if you want the test to fail in that case
+        // expect(report.length).toBe(0);
+    });
+});
```
Comment (tymmesyde) [inline tests/i18nScan.test.js]: Could we do something like this instead?
```suggestion
const report = [];
walk(directoryToScan, report);

describe.each(report)('hardcoded string', (entry) => {
    it(`should not have "${entry.string}" in ${entry.file} at line ${entry.line}`, () => {
        expect(entry.string).toBeFalsy();
    });
});
```

File: src/common/useTranslate.js (hunk 1 above)
Comment (Botsy) [reply src/common/useTranslate.js]: You are right. It was used for discover extra dropdown options, but we don't need it, so I will revert this change and fix the options for extra dropdowns.

File: tests/i18nScan.test.js (hunk 2 above)
Comment (Botsy) [reply tests/i18nScan.test.js]: Wouldn't failing of those tests affect the build process?

Comment (tymmesyde) [reply tests/i18nScan.test.js]: Yes, so we can enforce using translations strings when adding stuff to the codebase so we don't have to go back again like you did to add missing strings

**Review by tymmesyde — APPROVED** [review]
